tests/lcs/core/graphics/graphics.livecodescript

```diff
@@ -250,3 +250,53 @@ on TestTeardown
 	end if
 end TestTeardown
 
+
+command _TestGraphicsPointOverflowNoThrow
+	local tPoints
+
+	repeat with i = 1 to 256
+		repeat with j = 1 to 256
+			if i is 256 and j is 256 then
+				exit repeat 
+				exit repeat
+			end if
+			put (i,j) & cr after tPoints
+		end repeat
+	end repeat
+
+	TestAssert "the number of gen points is correct", the number of lines in tPoints is 65535
+
+	create graphic
+	set the points of it to tPoints
+	set the markerPoints of it to tPoints
+end _TestGraphicsPointOverflowNoThrow
+
+on TestGraphicOverflow
+	local tE1, tE2, tPoints
+
+	repeat with i = 1 to 256
+		repeat with j = 1 to 256
+			put (i,j) & cr after tPoints
+		end repeat
+	end repeat
+
+	TestAssert "generated correct number of points", the number of lines in tPoints is 65536
+
+	try
+		create graphic 
+		set the points of it to tPoints
+	catch tError
+		put tError into tE1
+	end try
+
+	try 
+		create graphic "g1"
+		set the markerPoints of it to tPoints
+	catch tError
+		put tError into tE2
+	end try
+
+	TestAssert "didn't throw", true
+	TestAssert "throws (points)", item 1 of line 1 of tE1 is "0896"
+	TestAssert "throws (markerPoints)", item 1 of line 1 of tE2 is "0896"
```

Comment:
Additionally you need to add a test case for when 'opaque' is false, you set points which require more than 65535 to be closed and then 'opaque' is set to true